### PR TITLE
add node example of ua connect api -> postgres

### PR DESCRIPTION
**Pull request by AWinterman**

### CODE REVIEW

File: connect-examples/node-connect-postgres/index.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+
+var config = {
+  user: 'urban_airship_connect',
+  password: process.env.CONNECT_PW,
+  database: 'urban_airship_connect',
+  host: process.env.PGHOST,
+  port: process.env.PGPORT,
+  query: process.env.UA_CONNECT_QUERY || {start: "EARLIEST"}, 
+  offsetCommitInterval: 1000 * 60 // every minute
```
Comment (AWinterman) [inline connect-examples/node-connect-postgres/index.js]: this is ignored currently

File: connect-examples/node-connect-postgres/package.json [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    "test": "echo \"Error: there are no tests\" && exit 1",
+    "user": "createuser -h $PGHOST -p $PGPORT -D -R urban_airship_connect",
+    "schema": "psql --dbname urban_airship_connect --host $PGHOST --port $PGPORT -U $PGUSER --file schema.sql",
+    "connect": "psql --dbname urban_airship_connect --host $PGHOST --port $PGPORT -U $PGUSER",
+    "start": "node index.js"
+  },
+  "author": "Urban Airship",
+  "license": "MIT",
+  "dependencies": {
+    "dotpath-stream": "^2.0.1",
```
Comment (AWinterman) [inline connect-examples/node-connect-postgres/package.json]: not used

File: connect-examples/node-connect-postgres/package.json [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+    "pg": "^6.1.5",
+    "pino": "^4.3.0",
```
Comment (AWinterman) [inline connect-examples/node-connect-postgres/package.json]: also not used but maybe i should?

File: connect-examples/node-connect-postgres/postgres.js [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+var through = require('through2').obj
+var xtend = require('xtend')
+
+module.exports = pgStream
+
+function pgStream (app, config, ready) {
+  var pool = new pg.Pool(config)
+
+  var UPDATE =  {
+    text: 'INSERT INTO apps(app, stream_offset) VALUES ($1, $2) ON CONFLICT (app) DO UPDATE SET stream_offset=$2 RETURNING (stream_offset)',
```
Comment (AWinterman) [inline connect-examples/node-connect-postgres/postgres.js]: Stores the offset, overwriting whatever is there. If it's the first go round we'll just write 0.

File: connect-examples/node-connect-postgres/package.json (hunk 3 above)
Comment (AWinterman) [reply connect-examples/node-connect-postgres/package.json]: (it's a thing for logging)

**Review by jarofghosts — APPROVED** [review]

**Comment by AWinterman** [comment]
@jarofghosts @fardog imma close this. You can recover and merge it if you want.

Also 👋

**Pull request closed without merging**